src/wp-includes/default-filters.php

```diff
@@ -357,6 +357,7 @@
 add_action( 'start_previewing_theme', 'wp_clean_theme_json_cache' );
 add_action( 'after_switch_theme', '_wp_menus_changed' );
 add_action( 'after_switch_theme', '_wp_sidebars_changed' );
+add_action( 'wp_enqueue_scripts', 'enqueue_emoji_styles' );
```

Comment:
I suggest adding an inline comment after the `print_emoji_styles` instance:

```php
add_action( 'wp_print_styles', 'print_emoji_styles' ); // Retained for backwards-compatibility. Unhooked by wp_enqueue_emoji_styles().
```

See precedent on this line:

https://github.com/WordPress/wordpress-develop/blob/05b09c3e9153d4e04a44d9fd4458e8c501e22df2/src/wp-includes/default-filters.php#L675